pallets/funding/src/runtime_api.rs

```diff
@@ -70,7 +70,22 @@ impl<T: Config> Pallet<T> {
 
 	pub fn top_bids(project_id: ProjectId, amount: u32) -> Vec<BidInfoOf<T>> {
 		Bids::<T>::iter_prefix_values((project_id,))
-			.sorted_by(|a, b| b.final_ct_amount.cmp(&a.final_ct_amount))
+			.sorted_by(|a, b| {
+				let a_final_ct_amount = match a.status {
```

Comment:
I've seen this match structure 4 times in this pr so might be nice to move this to a function in Bid impl.